src/ucp/rma/flush.c

```diff
@@ -42,19 +43,23 @@ static void ucp_ep_flush_progress(ucp_request_t *req)
 
     /* If the number of lanes changed since flush operation was submitted, adjust
      * the number of expected completions */
-    if (ucs_unlikely(req->send.flush.num_lanes != ucp_ep_num_lanes(ep))) {
-        diff                            = ucp_ep_num_lanes(ep) -
-                                          req->send.flush.num_lanes;
-        if (diff < 0) {
-            ucs_fatal("ep %p: unsupported endpoint reconfiguration from %d to %d"
-                      " lanes during flush", ep, req->send.flush.num_lanes,
-                      ucp_ep_num_lanes(ep));
+    if (ucs_unlikely(req->send.flush.num_lanes != num_lanes)) {
+        diff                      = num_lanes - req->send.flush.num_lanes;
+        req->send.flush.num_lanes = num_lanes;
+        if (diff >= 0) {
+            ucp_trace_req(req,
```

Comment:
maybe print it for error flow as well, because number of lanes is also changed (decreased)